lib/utils/answer_controller.dart

```diff
@@ -42,12 +51,14 @@ class AnswerController {
       SimpleLogger().shout(
         "Error while answering single select question, persisting for later...: $exception",
       );
-      await reportError(exception, stackTrace);
+      await reportError(
+          SurveyAnswerException(exception, answer: builtAnswer), stackTrace);
```

Comment:
Shouldn't these two be another way around? It would be nice that the answer would still be persisted even when the error reporting fails

Reply:
Resolved.